src/ui/public/styles/dark-theme.less

```diff
@@ -535,5 +572,9 @@
       }
     }
   }
+}
 
+::-webkit-scrollbar {
```

Comment:
Along the same line, can this be defined in both the dark-theme.less file and in the /dashboard/styles/main.less file?

There should be a variable added to /src/ui/public/styles/variables/for-theme.less and /src/ui/public/styles/dark-variables.less ie: (at)webkit-scrollbar-bg which has the value of darken(@panel-bg, 2%); and is consumed in both places.